packages/components/Button/src/Button.android.tsx

```diff
@@ -0,0 +1,82 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { Pressable, View } from 'react-native';
+import { IButtonSlotProps, IButtonState, IButtonProps, IButtonRenderData, buttonName, IButtonType } from './Button.types';
+import { compose, IUseComposeStyling } from '@uifabricshared/foundation-compose';
+import { ISlots, withSlots } from '@uifabricshared/foundation-composable';
+import { Text } from '@fluentui-react-native/text';
+import { settings } from './Button.settings';
+import { backgroundColorTokens, borderTokens, textTokens, foregroundColorTokens, getPaletteFromTheme } from '@fluentui-react-native/tokens';
+import { filterViewProps } from '@fluentui-react-native/adapters';
+import { mergeSettings } from '@uifabricshared/foundation-settings';
+import { useAsPressable, useViewCommandFocus } from '@fluentui-react-native/interactive-hooks';
+import { Icon } from '@fluentui-react-native/icon';
+import { createIconProps } from './Button.icon';
+
+export const Button = compose<IButtonType>({
```

Comment:
Is the difference between the base button and the android one so big that we actually need  a platform specific file? It will be hard to track differences between the two as they evolve.

Reply:
Yep, I was in a doubt regarding this, but since android button has an extra slot and thus, changed (and extra) slot settings; it was necessary for readability of code.